fdbclient/include/fdbclient/MetaclusterRegistration.h

```diff
@@ -28,30 +28,65 @@
 
 std::string clusterTypeToString(const ClusterType& clusterType);
 
-struct MetaclusterRegistrationEntry {
+enum class MetaclusterVersion {
```

Comment:
Since we will likely compare different versions as integers, is `enum MetaclusterVersion : uint8_t` more convenient than `enum class` with similar type-safety?

Reply:
We can compare the versions directly with each other, and `enum class` provides better type safety than the `enum` and avoids polluting the namespace